backend/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9.13
+
+WORKDIR /code
+
+COPY ./requirements.txt /code/requirements.txt
+
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+COPY ./app /code/app
+
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
+
+# If running behind a proxy like Nginx or Traefik add --proxy-headers
```

Comment:
If don't use this code, then remove it! Don't leave commented code never! If code has been commented, then it isn't need.